app/code/core/Mage/CatalogSearch/Model/Layer/Filter/Attribute.php

```diff
@@ -33,12 +33,11 @@ class Mage_CatalogSearch_Model_Layer_Filter_Attribute extends Mage_Catalog_Model
     /**
      * Check whether specified attribute can be used in LN
      *
-     * @param Mage_Catalog_Model_Resource_Eav_Attribute  $attribute
-     * @return bool
+     * @param Mage_Catalog_Model_Resource_Eav_Attribute $attribute
+     * @return Mage_Catalog_Model_Resource_Eav_Attribute
```

Comment:
bool was correct

Reply:
This is wrong (proberbly copied param to return), but ...

- in DB its tinynt/bool
- parrents `@method` say `int`
- in code ìnt` is used

I'll revert, but tis has to be chekced again. (W'd love to have strict types :P)